--- acpica-unix-20060912/compiler/Makefile	2007-01-16 01:00:50.313533493 +0200
+++ acpica-unix-20060912/compiler/Makefile	2007-01-16 01:04:50.238914927 +0200
@@ -110,7 +110,9 @@
 CLEANFILES= y.output y.tab.c y.tab.h aslcompiler.y.h \
 	aslcompilerparse.c aslcompilerlex.c iasl
 
-aslcompilerparse.c: aslcompiler.y
+aslanalyze.c aslcodegen.c aslcompilerlex.c aslfold.c asllength.c asllisting.c aslload.c asllookup.c aslopcodes.c asloperands.c aslopt.c aslresource.c aslrestype1.c aslrestype2.c asltransform.c asltree.c aslutils.c: aslcompiler.y.h
+
+aslcompilerparse.c aslcompiler.y.h: aslcompiler.y
 	${YACC} ${YFLAGS} aslcompiler.y
 	cp y.tab.c aslcompilerparse.c
 	cp y.tab.h aslcompiler.y.h
